daphne_server/src/router/test_routes.rs

```diff
@@ -145,6 +148,15 @@ async fn add_task(
     }
 }
 
+#[tracing::instrument(skip(app, cmd))]
+async fn add_task_default(
+    State(app): State<Arc<App>>,
+    Json(cmd): Json<InternalTestAddTask>,
+) -> impl IntoResponse {
+    let version = app.service_config.default_version;
+    add_task(State(app), Path(version), Json(cmd)).await
+}
+
```

Comment:
nit: No need to destructure the Json extractor if you're just going to pass it along

 ```rust
async fn add_task_default(
    State(app): State<Arc<App>>,
    cmd: Json<InternalTestAddTask>,
) -> impl IntoResponse {
    let version = app.service_config.default_version;
    add_task(State(app), Path(version), cmd).await
}
```

Reply:
Done.